docs-app/.prettierrc.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = {
+  plugins: ['prettier-plugin-ember-template-tag'],
```

Comment:
!! Should we be using this in our other packages? Does it format everything in between `<template>` tags (is this what it does?)

Reply:
We have this already in the test-app for example. It makes prettier even understand the existence of `<template>`, otherwise it wouldn't be able to parse the file (as it's not valid JS/TS). And yes, it does also format everything inside of the template!